docs/pages/management/export-audit-events/panther.mdx

```diff
@@ -0,0 +1,226 @@
+description: How to configure the Teleport Event Handler plugin and Fluentd to send audit logs to Panther
+---
+
+Panther is a cloud-native security analytics platform. In this guide, we'll explain
+how to forward Teleport audit events to Panther using Fluentd.
+
+The Teleport Event Handler is designed to communicate with Fluentd using mTLS
+to establish a secure channel. In this setup, the Event Handler sends events to Fluentd, which forwards them to S3 to be ingested by Panther.
+
+![Architecture of the setup shown in this guide](../../../img/management/fluentd-diagram.png)
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/machine-id/plugin-prerequisites.mdx!)
+
+- A [Panther](https://panther.com/product/request-a-demo/) account.
+- A server, virtual machine, Kubernetes cluster, or Docker environment to run the
+  Event Handler. The instructions below assume a local Docker container for testing.
+- Fluentd version v(=fluentd.version=) or greater. The Teleport Event Handler
+  will create a new `fluent.conf` file you can integrate into an existing Fluentd
+  system, or use with a fresh setup.
+- An S3 bucket to store the logs. Panther will ingest the logs from this bucket.
+
+The instructions below demonstrate a local test of the Event Handler plugin on
+VM. You will need to adjust paths, ports, and domains for other environments.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/6. Install the Event Handler plugin
+
+The Teleport event handler runs alongside the Fluentd forwarder, receives events
+from Teleport's events API, and forwards them to Fluentd.
+
+(!docs/pages/includes/install-event-handler.mdx!)
+
+## Step 2/6. Generate a plugin configuration
+
+(!docs/pages/includes/configure-event-handler.mdx!)
+
+## Step 3/6. Create a user and role for reading audit events
+
+(!docs/pages/includes/plugins/event-handler-role-user.mdx!)
+
+## Step 4/6. Create teleport-event-handler credentials
+
+### Enable impersonation of the Event Handler user
+
+(!docs/pages/includes/plugins/rbac-impersonate-event-handler.mdx!)
+
+### Export an identity file for the Event Handler plugin user
+
+Give the plugin access to a Teleport identity file. We recommend using Machine
+ID for this in order to produce short-lived identity files that are less
+dangerous if exfiltrated, though in demo deployments, you can generate
+longer-lived identity files with `tctl`:
+
+<Tabs>
+<TabItem label="Machine ID">
+(!docs/pages/includes/plugins/tbot-identity.mdx secret="teleport-event-handler-identity"!)
+</TabItem>
+<TabItem label="Long-lived identity files">
+(!docs/pages/includes/plugins/identity-export.mdx user="teleport-event-handler" secret="teleport-event-handler-identity"!)
+</TabItem>
+</Tabs>
+
+## Step 5/6. Create a Dockerfile with Fluentd and the S3 plugin
+
+To send logs to Panther, you need to use the Fluentd output plugin for S3. Create
+a `Dockerfile` with the following content:
+
+```Dockerfile
```

Comment:
The text looks a little faded in the rendered page—should we use `text` here?

Reply:
Updated